client/components/chart/style.scss

```diff
@@ -0,0 +1,222 @@
+/** @format */
+.woocommerce-dashboard__widget-chart {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: flex-start;
+	margin: -20px;
+	border-top: 0.5px solid rgba(226, 228, 231, 0.7);
```

Comment:
Do we already have colors defined that will match these? If not, should we add a scss variable color for this?